app/controllers/hyrax/file_sets_controller.rb

```diff
@@ -254,12 +225,6 @@ def empty_file?(file)
         (file.respond_to?(:tempfile) && file.tempfile.size == 0) || (file.respond_to?(:size) && file.size == 0)
       end
 
-      # this is provided so that implementing application can override this behavior and map params to different attributes
```

Comment:
@mjgiarlo The code you link to is all forked anyway:
- `ihrnexuslab`'s `update_metadata_from_upload_screen` requires a param!  That means they are already overriding the method that invokes it, too (and not calling `super`).  
- `sephirothkod`: same thing
- `HyrdraNorth` decided everything in their `FilesControllerBehavior` including this method would be `protected`... via a module, no less!  
- Duke has the cleanest implementation, but makes the method `public`.  Like others, it has decided an instance variable named `@generic_file` means something.  But it doesn't in Hyrax.